jerry-main/main-unix.c

```diff
@@ -107,6 +107,21 @@ assert_handler (const jerry_value_t func_obj_val __attribute__((unused)), /**< f
   }
 } /* assert_handler */
 
+/**
+ * Provide the '__gc' implementation for the engine.
```

Comment:
This is not the first time I needed a gc call, and I finally decided to create one. I chose `__gc()` because `gc()` is too short and might be defined by some tests. We don't need to follow other engines since this is our test utility.